app/helpers/application_helper.rb

```diff
@@ -41,6 +41,10 @@ def icon(name)
     image_tag "icons/#{name}.png"
   end
 
+  def ldate(date, options = {})
```

Comment:
I agree on having a more descriptive name, **but** it's more a shorthand akin to `I18n.l` aliasing `I18n.localize`.
Do you want to keep the informative/verbose `I18n.t(:date_not_set)`? Otherwise rejecting this PR and making a new one by just introducting `#event_start_time` on the other view might be a simpler approach.

On a side note: Whenever i see `#model_name_attribute` on a helper i ask myself if a presenter (or viewmodel) wouldn't be more appropriate. Taking a look at `Event` itself reveals several code smells of a presenter object which tries to escape:
* `Event#humanize_time_str` - view represenation of some time attribute shouldn't be this models concern
* `Event#track_name` - reaching "through" to another object with a `#try` should be a presenters task unless we use it for internal logic
* `Event#feedback_standard_deviation` - not actually computing a number, but a string for representation purpose
* `Event#slug` - same here, this is a presenter's work.

Including `event_start_time` this gives good enough reason for such an object. One would have to take a look if  cases on other models exist, so we can pull a dependency in for doing this properly.